CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [TBD]
+### Changed
+* Enable federated metastore access-control-type to be configured to READ_ONLY, READ_AND_WRITE_AND_CREATE, READ_AND_WRITE_ON_DATABASE_WHITELIST or READ_AND_WRITE_AND_CREATE_ON_DATABASE_WHITELIST
```

Comment:
we probably need to decide how to handle `create database`, couple of options:
1) only allow READ_ONLY and READ_AND_WRITE_ON_DATABASE_WHITELIST (perhaps we add a READ_AND_WRITE?)
2) we agree on a strategy for creating db's. PrefixMode we follow prefixes (so we create in remote if it has the correct prefix (transforming the db name along the way), ManualMode we only create db's in primary.

NOTE: and access control strategy that says WRITE allows for `drop database` statements to be performed.

Reply:
Should we do this in another ticket or tackle it now? I feel like we will want some consensus on how we should handle it